cloudify/test_utils/local_workflow_decorator.py

```diff
@@ -69,7 +69,7 @@ def _expand_dictionary(inputs, func_self, func_args, func_kwargs):
     func_to_call = None
     if callable(inputs):
         func_to_call = inputs
-    elif isinstance(inputs, basestring):
+    elif isinstance(inputs, str):
```

Comment:
Is there a reason this doesn't use text_type- just because it's a test or?

Reply:
eh, might as well leave it as "native str" here, it's just a test util, and used 1 time, too

Reply:
(yes- resolved)